src/simulator/layer.h

```diff
@@ -47,7 +47,8 @@ internal bool layer_link(State* state,
                          SynapseCls* cls, f32 weight); 
 
 // NOTE: to set inputs
-internal void layer_step(Layer* layer, u32 time);
```

Comment:
I think we could just reaceive in every layer a pool of threads (the same one) and don't complicate these functions